🐛Issue with adding liquid-auth-js to Next ts project

**Describe the bug**

I tried installing the liquid-auth-js using npm on a new next ts project and got the following error after inserting the code from the [guide](https://github.com/algorandfoundation/liquid-auth-js) into the page.tsx .

`import {SignalClient} from '@algorandfoundation/liquid-client';`
`const client = new SignalClient(window.origin);`

The error I got was "Module not found: Default condition should be last one"

**To Reproduce**

1. Initialize new Next ts project
2. Run `npm install algorandfoundation/liquid-auth-js --save`
3. Insert the above code into page.tsx
4. Start Next ts server


**Screenshots**
![Screenshot 2024-08-22 at 1 25 42 PM](https://github.com/user-attachments/assets/4e76d27b-a52b-4518-93c7-55a79fe25d6e)


**Desktop (please complete the following information):**
 - OS: IOS
 - Browser Chrome
 - Next version:  14.2.5
 - Node version: 20.10.0


**Additional context**
Add any other context about the problem here.

--- a/package.json
+++ b/package.json
@@ -6,28 +6,28 @@
   "type": "module",
   "exports": {
     ".": {
-      "default": "./lib/index.js",
-      "types": "./lib/index.d.ts"
+      "types": "./lib/index.d.ts",
+      "default": "./lib/index.js"
     },
     "./attestation": {
-      "default": "./lib/attestation.js",
-      "types": "./lib/attestation.d.ts"
+      "types": "./lib/attestation.d.ts",
+      "default": "./lib/attestation.js"
     },
     "./assertion": {
-      "default": "./lib/assertion.js",
-      "types": "./lib/assertion.d.ts"
+      "types": "./lib/assertion.d.ts",
+      "default": "./lib/assertion.js"
     },
     "./signal": {
-      "default": "./lib/signal.js",
-      "types": "./lib/signal.d.ts"
+      "types": "./lib/signal.d.ts",
+      "default": "./lib/signal.js"
     },
     "./errors": {
-      "default": "./lib/errors.js",
-      "types": "./lib/errors.d.ts"
+      "types": "./lib/errors.d.ts",
+      "default": "./lib/errors.js"
     },
     "./encoding": {
-      "default": "./lib/encoding.js",
-      "types": "./lib/encoding.d.ts"
+      "types": "./lib/encoding.d.ts",
+      "default": "./lib/encoding.js"
     }
   },
   "scripts": {

